src/gui/wallet.go

```diff
@@ -269,7 +270,7 @@ func walletNewAddresses(gateway *daemon.Gateway) http.HandlerFunc {
 }
 
 // Update wallet label
-func walletUpdateHandler(gateway *daemon.Gateway) http.HandlerFunc {
+func walletUpdateHandler(gateway Gatewayer) http.HandlerFunc {
```

Comment:
Can you make this method POST-only

and add documentation for the endpoint to gui/README.md